modules/notifications/notifications.php

```diff
@@ -1,48 +1,34 @@
 <?php
 /**
- * class Notifications
- * Email notifications for VIP Workflow and more
+ * The notifications module for sending email/webhook notifications.
+ *
+ * @package vip-workflow
  */
 namespace VIPWorkflow\Modules;
 
 use VIPWorkflow\VIP_Workflow;
-use VIPWorkflow\Modules\Shared\PHP\Module;
 use function VIPWorkflow\Modules\Shared\PHP\vw_draft_or_post_title;
 
-class Notifications extends Module {
-
-	public $module;
-
-	/**
-	 * Register the module with VIP Workflow but don't do anything else
-	 */
-	public function __construct() {
-
-		// Register the module with VIP Workflow
-		$this->module_url = $this->get_module_url( __FILE__ );
-		$args             = [
-			'module_url'            => $this->module_url,
-			'slug'                  => 'notifications',
-		];
-		$this->module     = VIP_Workflow::instance()->register_module( 'notifications', $args );
-	}
+class Notifications {
 
 	/**
-	 * Initialize the notifications class if the plugin is enabled
+	 * Initialize the notifications module
 	 */
-	public function init() {
+	public static function init(): void {
 		// Send notifications on post status change
-		add_action( 'transition_post_status', [ $this, 'notification_status_change' ], 10, 3 );
+		add_action( 'transition_post_status', [ __CLASS__, 'notification_status_change' ], 10, 3 );
+
 		// Schedule email sending
-		add_action( 'vw_send_scheduled_emails', [ $this, 'send_emails' ], 10, 4 );
+		add_action( 'vw_send_scheduled_emails', [ __CLASS__, 'send_emails' ], 10, 4 );
+
 		// Schedule webhook sending
-		add_action( 'vw_send_scheduled_webhook', [ $this, 'send_to_webhook' ], 10, 4 );
+		add_action( 'vw_send_scheduled_webhook', [ __CLASS__, 'send_to_webhook' ], 10, 4 );
 	}
 
 	/**
 	 * Set up and send post status change notification email
 	 */
-	public function notification_status_change( $new_status, $old_status, $post ) {
+	public static function notification_status_change( string $new_status, string $old_status, \WP_Post $post ): void {
```

Comment:
`\WP_Post` works fine, but we can also `use WP_Post;` at the top of this file and the backslash would no longer be necessary.

Reply:
If you fix this, also see function signatures for `schedule_emails()` and `get_scheduled_datetime()`

Reply:
Good catch, fixed all this up